test/units/module_utils/facts/system/distribution/fixtures/rockylinux_8_3.json

```diff
@@ -0,0 +1,46 @@
+{
+    "name": "Rocky 8.3",
+    "distro": {
+        "codename": "",
+        "id": "rocky",
+        "name": "Rocky Linux",
+        "version": "8.3",
```

Comment:
In my testing `distro.version()` returns `8` not `8.3`. This means `result['distribution_version']` will be `8` and not `8.3`.

```
{
    "name": "Rocky 8",
    "distro": {
        "codename": "",
        "id": "rocky",
        "name": "Rocky Linux",
        "version": "8",
        "version_best": "8.3",
        "lsb_release_info": {},
        "os_release_info": {
            "name": "Rocky Linux",
            "version": "8",
            "id": "rocky",
            "id_like": "rhel fedora",
            "version_id": "8",
            "platform_id": "platform:el8",
            "pretty_name": "Rocky Linux 8",
            "ansi_color": "0;31",
            "cpe_name": "cpe:/o:rocky:rocky:8",
            "home_url": "https://rockylinux.org/",
            "bug_report_url": "https://bugs.rockylinux.org/",
            "rocky_support_product": "Rocky Linux",
            "rocky_support_product_version": "8"
        }
    },
    "input": {
        "/etc/redhat-release": "Rocky Linux release 8.3\n",
        "/etc/system-release": "Rocky Linux release 8.3\n",
        "/etc/os-release": "NAME=\"Rocky Linux\"\nVERSION=\"8\"\nID=\"rocky\"\nID_LIKE=\"rhel fedora\"\nVERSION_ID=\"8\"\nPLATFORM_ID=\"platform:el8\"\nPRETTY_NAME=\"Rocky Linux 8\"\nANSI_COLOR=\"0;31\"\nCPE_NAME=\"cpe:/o:rocky:rocky:8\"\nHOME_URL=\"https://rockylinux.org/\"\nBUG_REPORT_URL=\"https://bugs.rockylinux.org/\"\nROCKY_SUPPORT_PRODUCT=\"Rocky Linux\"\nROCKY_SUPPORT_PRODUCT_VERSION=\"8\"\n",
        "/usr/lib/os-release": "NAME=\"Rocky Linux\"\nVERSION=\"8\"\nID=\"rocky\"\nID_LIKE=\"rhel fedora\"\nVERSION_ID=\"8\"\nPLATFORM_ID=\"platform:el8\"\nPRETTY_NAME=\"Rocky Linux 8\"\nANSI_COLOR=\"0;31\"\nCPE_NAME=\"cpe:/o:rocky:rocky:8\"\nHOME_URL=\"https://rockylinux.org/\"\nBUG_REPORT_URL=\"https://bugs.rockylinux.org/\"\nROCKY_SUPPORT_PRODUCT=\"Rocky Linux\"\nROCKY_SUPPORT_PRODUCT_VERSION=\"8\"\n"
    },
    "platform.dist": [
        "rocky",
        "8",
        ""
    ],
    "result": {
        "distribution": "Rocky",
        "distribution_version": "8",
        "distribution_release": "NA",
        "distribution_major_version": "8",
        "os_family": "Rocky"
    },
    "platform.release": "4.18.0-240.22.1.el8.x86_64"
}
```

`rocky` needs to be added to the list of distros that need the more precise version information, and the fixture needs updating.
https://github.com/ansible/ansible/blob/d8c20a73a4c3d7d0383167a2c50dd077d15b1ba8/lib/ansible/module_utils/common/sys_info.py#L53-L56

This conditional needs to be updated as well.
https://github.com/ansible/ansible/blob/d8c20a73a4c3d7d0383167a2c50dd077d15b1ba8/lib/ansible/module_utils/common/sys_info.py#L63-L76

Reply:
This does not need further changes if you update your release to the latest version. Please update to the [latest version](https://kojidev.rockylinux.org/koji/buildinfo?buildID=7923) if you wish. We have not pushed updates to our repos yet.

Thank you.

Reply:
Awesome! Thanks. I just tested with a recent RC, but evidently it was not the latest.

Reply:
I tested with `rocky-release-8.3-13.el8.noarch` and that gives the expected output.